speechmatics/models.py

```diff
@@ -8,6 +8,16 @@
 
 
 def del_none(dictionary):
+    """
+    Recursively delete all keys from the dictionary which are pointing to
```

Comment:
Don't like the word `pointing`. How about
```
Recursively delete all keys from the dictionary which values are None.
```

Reply:
Yes, that is better, thanks!

"Value of the key" is a bit ambiguous. Does "Recursively delete from the dictionary all _entries_ which values are None."  sound okay?

Reply:
sounds good